File: guanabara-java/src/com/danillo/algoritmos/lista04/Exercicio50.java
package com.danillo.algoritmos.lista04;

import java.util.Random;

public class Exercicio50 {
    public static void main(String[] args) {
        int i = 0;
        int acimaDeCinco = 0;
        int divisivelPorTres = 0;

        while(i < 5){
            Random random = new Random();

            int aleatorio = random.nextInt(10);
            System.out.println(aleatorio);

            if(aleatorio > 5){
                acimaDeCinco++;
            }

            if(aleatorio % 3 == 0){
                divisivelPorTres++;
            }

            i++;
        }
        System.out.println("Números maiores que 5: " +acimaDeCinco+ "\nNúmeros divisíveis por 3: " +divisivelPorTres);
    }
}

File: guanabara-python/src/lista04/exercicio50.py
import random

i = 0
maior_5 = 0
divisivel_3 = 0

while i < 20:
    x = random.randint(1, 20)
    print(x)

    if x > 5:
        maior_5 = maior_5 + 1

    if x % 3 == 0:
        divisivel_3 = divisivel_3 + 1

    i = i + 1

print(f"Quantidade de números sorteados: {i}\nNúmeros acima de 5: {maior_5}\nNúmeros divisíveis por 3: {divisivel_3}")
